.github/workflows/release.yml

```
@@ -1,16 +1,33 @@
 
 permissions:
   contents: write
 
 jobs:
-  build:
+  build-signed-macos-app:
     runs-on: macos-latest
+    timeout-minutes: 60
+    env:
+      APP_NAME: Typeflux
+      APPLE_CERTIFICATE_PATH: ${{ runner.temp }}/developer-id-application.p12
```

actionlint (1.7.12)

[error] context "runner" is not allowed here. available contexts are "github", "inputs", "matrix", "needs", "secrets", "strategy", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)